certbot/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Added
 
-*
+* Updated Apache/NGINX TLS configs to show contents are based on ssl-config.mozilla.org
```

Comment:
I'd like to tweak this wording slightly, but otherwise this LGTM. Thanks for the PR!
```suggestion
* Updated Apache/NGINX TLS configs to document contents are based on ssl-config.mozilla.org
```